springbootsample/README.md

```diff
@@ -0,0 +1,21 @@
+# Run Application with SpringBoot Application
+
+This is very simple example demonstrating how to run instrument Springboot application with Azure Application Insights
+Springboot Starter and Application Insights Java Agent to track dependencies.
+
+
+### How to build the application?
```

Comment:
minor: "Build"

Reply:
My understanding is this and the previous comment was an ask for capitalization.

Reply:
I suggest we rename this phrase "How to build the application?" to "Build"

Reply:
Done